lib/resources/dstu2/patient.yaml

```diff
@@ -0,0 +1,219 @@
+---
+name: Patient
+field_name_base_url: http://hl7.org/fhir/DSTU2/patient-definitions.html#Patient
+fields:
+- name: identifier
+  required: 'No'
+  cardinality: 0..*
+  type: Identifier
+  description: The identifier for the patient.
+  children:
+
+  - name: use
+    required: 'No'
+    cardinality: 0..1
+    type: code
+    description: The purpose of the identifier.
+    binding:
```

Comment:
For the terminology tables we are only documenting the fields with Preferred or Example binding strengths (or possibly Extensible if our implementation deviates from the defined valueset).

It looks like the only field which might need to be documented would be Patient.contact.relationship, and then only if we have some Cerner specific values we are returning.

@daliboz Can you confirm before Zameer yanks on these out?